src/compiler/program.ts

```diff
@@ -329,6 +329,7 @@ namespace ts {
         let files: SourceFile[] = [];
         let fileProcessingDiagnostics = createDiagnosticCollection();
         let programDiagnostics = createDiagnosticCollection();
+        let emitBlockingDiagnostics = createDiagnosticCollection();

```

Comment:
Is it really ok to capture? If the options change (from the language service, would we not need to calculate it again?)